retail/retail-java-applications/monitoring/dashboards/README.MD

```diff
@@ -0,0 +1,44 @@
+# Using Terraform to deploy monitoring dashboards
```

Comment:
Could we move this folder up one under retail rather than retail-java-applications? We plan on building a python version of this application.

Reply:
Have updated the PR with the new directory structure.